frontend/src/pages/Graphing/components/Graph.tsx

```diff
@@ -338,6 +341,8 @@ export const getViewConfig = (
 	return viewConfig
 }
 
+const POLL_INTERVAL_VALUE = 60000
```

Comment:
Currently set this to 1 minute - I don't think it makes sense to go less than this since we are using the start the minutes and end of the minute to fetch the metric data, resulting in the cache being hit for fetches within the minute.

We could look to increase this for longer time frames, but think it is still useful to see up-to-date data

Reply:
Not a huge deal, but I think a longer poll interval would be nice for long time periods (6+ hours) because the graph won't change much (each bucket size is likely 5+ minutes) and it will add extra load on clickhouse. Separately, is there anything easy we could do to avoid refreshing while a user is interacting with the graph?

Reply:
Increased this interval to 5 minutes for data showing 6+ hours

Reply:
> Separately, is there anything easy we could do to avoid refreshing while a user is interacting with the graph?

I'd have to look more into this. I don't have a great method off the top of my head, but we could disable the polling on the edit page?